cns/configuration/cns_config.json

```diff
@@ -1,23 +1,24 @@
 {
     "TelemetrySettings": {
```

Comment:
Why so many changes? Is it not rebased to master?

Reply:
only change is putting the fields in lexical order